test/integration/members.test.js

```diff
@@ -156,31 +151,32 @@ describe("Members", function () {
         });
     });
 
-    it("Get all the idle members in the database", function (done) {
-      chai
-        .request(app)
-        .get("/members/idle")
-        .end((err, res) => {
-          if (err) {
-            return done(err);
-          }
+    it("Should return all the idle members", function (done) {
+      addUser().then(() => {
```

Comment:
Can we set this in separate `describe` in a `before`  as we discussed earlier for previous comments ?

Reply:
Chai recommends that if a describe has only 1 it block better don't use describe for it. It gives lint error also

Reply:
Updated the above comment as well
https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/no-hooks-for-single-case.md